src/types.ts

```diff
@@ -370,6 +370,7 @@ export interface CaaSApi_Section {
   identifier: string
   template: CaaSApi_Template
   formData: CaaSApi_DataEntries
+  displayed?: boolean
```

Comment:
Why is this optional? All other attributes always exist. Wouldn't it be better to ensure this attribute always exists with a default value (true?) if it is ever missing in the CaaS?

In the other types as well